aiida/orm/implementation/entities.py

```diff
@@ -0,0 +1,354 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""Classes and methods for backend non-specific entities"""
+
+import abc
+
+from aiida.orm.utils.node import clean_value, validate_attribute_extra_key
+
+
+class AttributesBackendEntity(abc.ABC):
+    """Mixin class that adds all methods for the attributes column to an entity"""
+
+    _dbmodel = None
+    dbmodel = None
```

Comment:
Hmmm, wonder if this is how we should do it. The `_dbmodel` will indeed refer to the actual DbModel instance (wrapped in the ModelWrapper), but `dbmodel` is the property defined in `BackendEntity`. I don't think we should be setting it to `None`. This might get complaints from `pylint` but that is just because it doesn't understand mixins. So I would remove `dbmodel = None` declaration